src/services/database_service.py

```diff
@@ -0,0 +1,103 @@
+import faiss
```

Comment:
we don't have a database but an index

Reply:
I can change the name, but we have a vector database.

Reply:
correct if i'm wrong but in my view, a database stays after i stop the application. we have an index built when running.
of course, not something crucial feel free to ignore this comment